### ISSUE TITLE
Definition of view in Hints

### ISSUE THREAD

**Issue by MilennialZero** [archimatetool/archi#389]
In the Hints panel when focus is the diagram, we see the definiition:
**_"A view is a viewpoint on a model"._**
Not sure this is an appropriate definition. Seems self-referencing.
Don't find that definition anywhere else in the Archimate literature. 

![definition of view](https://user-images.githubusercontent.com/30424386/44885300-640ed180-ad03-11e8-8af7-1793b37d1b74.png)

**Comment by jbsarrodie** [comment]
You're right. I'll update the definition with the best one from the standard or official glossary.

**Comment by jbsarrodie** [comment]
From the official glossary:
>A view is a part of an Architecture Description that addresses a set of related concerns and is tailored for specific stakeholders.

The standard also adds (14.3):
>A view is specified by means of a viewpoint, which prescribes the concepts, models, analysis techniques, and visualizations that are provided by the view.

So I suggest to replace the hint by:
>A view is a part of an Architecture Description that addresses a set of related concerns and is tailored for specific stakeholders.
>
>A view is specified by means of a viewpoint, which prescribes the concepts, models, analysis techniques, and visualizations that are provided by the view.
>
>The same elements from the Model Tree can be re-used multiple times in the same or different View.

**Comment by Phillipus** [comment]
Changed to suggested text.
